src/org/thoughtcrime/securesms/geolocation/DcLocation.java

```diff
@@ -4,11 +4,6 @@
 
 import java.util.Observable;
 
-
-/**
- * Created by cyberta on 06.03.19.
- */
-
```

Comment:
in general, we avoid such boilerplates - they get outdated etc, the git history is a much better and cheaper source